src/workloads/transactions/LLTAnalytics.yml

```diff
@@ -0,0 +1,284 @@
+      OperationOptions:
+        Upsert: false
+        WriteConcern:
+          Level: majority
+        Hint: caid_1_price_1_cuid_1
+
+  # Scanner
+  SnapshotScannerShortDuration: &SnapshotScannerShortDuration 1 minutes
+  SnapshotScannerMediumDuration: &SnapshotScannerMediumDuration 10 minutes
+  SnapshotScannerLongDuration: &SnapshotScannerLongDuration 60 minutes
+
+ActorTemplates:
+  - TemplateName: UpdateTemplate
+    Config:
+      Name: {^Parameter: {Name: "Name", Default: "Short.Update.Baseline"}}
+      Type: CrudActor
+      Threads: *ThreadsValue
+      Phases:
+        OnlyActiveInPhases:
+          Active: {^Parameter: {Name: "Active", Default: [5]}}
+          NopInPhasesUpTo: *MaxPhases
+          PhaseConfig:
+            GlobalRate: *GlobalRateValue
+            Threads: *ThreadsValue
+            CollectionCount: *CollectionCount
+            Database: *dbname
+            Duration: {^Parameter: {Name: "Duration", Default: *SnapshotScannerShortDuration}}
+            Blocking:  {^Parameter: {Name: "Blocking", Default: yes}}
+            Operations:
+              - *PtcUpdateOperation
+              - *PcUpdateOperation
+              - *CpcUpdateOperation
+              - *PtcUpdateOperation
+              - *PcUpdateOperation
+              - *CpcUpdateOperation
+              - *PtcUpdateOperation
+              - *PcUpdateOperation
+              - *CpcUpdateOperation
+              - *PtcUpdateOperation
+              - *PcUpdateOperation
+              - *CpcUpdateOperation
+              - *PtcUpdateOperation
+              - *PcUpdateOperation
+              - *CpcUpdateOperation
+              - *PtcUpdateOperation
+              - *PcUpdateOperation
+              - *CpcUpdateOperation
+              - *PtcUpdateOperation
+              - *PcUpdateOperation
+              - *CpcUpdateOperation
+              - *PtcUpdateOperation
+              - *PcUpdateOperation
+              - *CpcUpdateOperation
+
+  - TemplateName: ScanTemplate
+    Config:
+      Name: {^Parameter: {Name: "Name", Default: "Short.Scan.Snapshot"}}
+      Type: CollectionScanner
+      Threads: *CollectionCount
+      CollectionCount: *CollectionCount
+      Database: *dbname
+      GenerateCollectionNames: true
+      Phases:
+        OnlyActiveInPhases:
+          Active: {^Parameter: {Name: "Active", Default: [7]}}# [7]
+          NopInPhasesUpTo: *MaxPhases
+          PhaseConfig:
+            Duration: {^Parameter: {Name: "Duration", Default: *SnapshotScannerShortDuration}}
+            #ScanDuration: {^Parameter: {Name: "ScanDuration", Default: *SnapshotScannerShortDuration}}
+            ScanType: snapshot
+            #ScanContinuous: true
+            # GenerateCollectionNames: true
+            AggregatePipeline:
+              {array: [{"$match":{"cuid":{"$gte":500}}},
+                {"$unwind": "$orders"},
+                {"$group": {"_id": "$ts", "total": {"$sum":{ $multiply: [ "$orders.unitPrice", "$orders.quantity" ] }}}},
```

Comment:
grouping on ts is going to give you some odd and large groups (I suspect). Would it make more sense to group by something like cuid?

What does the output of the aggregation actually look like?

Reply:
The output of aggregation looks like:
{ "_id" : ISODate("2021-06-11T20:36:16.838Z"), "total" : 1231829.2169177427 }
{ "_id" : ISODate("2021-06-11T20:36:16.839Z"), "total" : 1616938.9089608237 }
{ "_id" : ISODate("2021-06-11T20:36:16.840Z"), "total" : 1205669.5251776665 }
{ "_id" : ISODate("2021-06-11T20:36:16.841Z"), "total" : 1792501.6233907759 }
{ "_id" : ISODate("2021-06-11T20:36:16.842Z"), "total" : 2011877.5808415299 }
{ "_id" : ISODate("2021-06-11T20:36:16.843Z"), "total" : 1916505.504180574 }
Not too nice. Ideally it should be per document - say id, data, etc and total - but I didn't have time to figure it out yet.